media-tv/kodi/kodi-17.3-r1.ebuild

```diff
@@ -0,0 +1,285 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Is there a reason to keep -r0 here? i.e. it is very likely that -r1 will be broken for someone?